### PR TITLE
Upgrade ESLint with fixes.

### PR DESCRIPTION
**Pull request by nosferatu500**
**What was fixed?**

[jsx-no-bind](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md) (Example: 3)

### CODE REVIEW

**Comment by mifi** [comment]
I'm sorry but I don't understand anything about what this PR will improve, care to explain?

**Comment by nosferatu500** [comment]
By original:
```
function onPreserveMovDataHelpPress() {
     toast.fire({ icon: 'info', timer: 10000, text: i18n.t('Preserve all MOV/MP4 metadata tags (e.g. EXIF, GPS position etc.) from source file? Note that some players have trouble playing back files where all metadata is preserved, like iTunes and other Apple software') });
}
```

Fixed:
```
const onPreserveMovDataHelpPress = () => {
     toast.fire({ icon: 'info', timer: 10000, text: i18n.t('Preserve all MOV/MP4 metadata tags (e.g. EXIF, GPS position etc.) from source file? Note that some players have trouble playing back files where all metadata is preserved, like iTunes and other Apple software') });
};
   ```
   
By original design, the function will be recreated every time the ```ExportConfirm.jsx``` component is rendered.

**Comment by mifi** [comment]
but `const onPreserveMovDataHelpPress = () => {` will also recreate the arrow function every time it's rendered, no?

**Comment by nosferatu500** [comment]
Yes, you're also right. I forgot to wrap it with useCallback. My bad...

File: .eslintrc [hunk 1]
```diff
@@ -1,8 +1,14 @@
 {
   "extends": [
     "airbnb",
-    "airbnb/hooks"
+    "airbnb/hooks",
+    "plugin:react/recommended",
```
Comment (mifi) [inline .eslintrc:5]: I'm not sure which value these added "extends" give.

File: .eslintrc [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+    "plugin:react/jsx-runtime",
+    "plugin:import/errors",
+    "plugin:import/warnings",
+    "plugin:react-hooks/recommended"
   ],
+  "plugins": ["react", "import"],
```
Comment (mifi) [inline .eslintrc:11]: and this (already inherited from airbnb, no?)

File: .eslintrc [hunk 3]
```diff
@@ -22,6 +28,8 @@
     "arrow-parens": 0,
     "jsx-a11y/control-has-associated-label": 0,
     "react/prop-types": 0,
-    "no-multiple-empty-lines": ["error", { "max": 2, "maxBOF": 0, "maxEOF": 0 }]
+    "no-multiple-empty-lines": ["error", { "max": 2, "maxBOF": 0, "maxEOF": 0 }],
+    "no-unused-vars": 0,
```
Comment (mifi) [inline .eslintrc:32]: why are these disabled?

File: package.json [hunk 4]
```diff
@@ -13,7 +13,7 @@
     "build": "yarn icon-gen && react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint --ext .jsx --ext .js .",
+    "lint": "eslint --fix '{src,public}/**/*.{js,jsx}'",
```
Comment (mifi) [inline package.json:16]: why are you changing this?

**Review by mifi — CHANGES_REQUESTED** [review]
see additional comments

File: .eslintrc (hunk 1 above)
Comment (nosferatu500) [reply .eslintrc:5]: It seems to be really nothing.

File: .eslintrc (hunk 2 above)
Comment (nosferatu500) [reply .eslintrc:11]: yep.

File: .eslintrc (hunk 3 above)
Comment (nosferatu500) [reply .eslintrc:32]: In some cases ESLInt returns that React is not being used, but it is not.

**Pull request merged**